src/assets/i18n/cs.json5

```diff
@@ -4077,39 +5246,82 @@
+  // TODO New key - Add a translation
+  "search.filters.applied.f.title": "Title",
   // "search.filters.applied.f.author": "Author",
   "search.filters.applied.f.author": "Autor",
   // "search.filters.applied.f.dateIssued.max": "End date",
   "search.filters.applied.f.dateIssued.max": "Datum ukončení",
   // "search.filters.applied.f.dateIssued.min": "Start date",
   "search.filters.applied.f.dateIssued.min": "Datum zahájení",
   // "search.filters.applied.f.dateSubmitted": "Date submitted",
-  "search.filters.applied.f.dateSubmitted": "Datum předložení",
+  "search.filters.applied.f.dateSubmitted": "Date vložení",
   // "search.filters.applied.f.discoverable": "Non-discoverable",
-  "search.filters.applied.f.discoverable": "Soukromé",
+  // TODO Source message changed - Revise the translation
+  "search.filters.applied.f.discoverable": "Nedohledatelné",
   // "search.filters.applied.f.entityType": "Item Type",
-  "search.filters.applied.f.entityType": "Typ položky",
-  // "search.filters.applied.f.has_content_in_original_bundle": "Contain Files",
-  "search.filters.applied.f.has_content_in_original_bundle": "Obsahuje soubory",
+  "search.filters.applied.f.entityType": "Typ záznamu",
+  // "search.filters.applied.f.has_content_in_original_bundle": "Has files",
+  "search.filters.applied.f.has_content_in_original_bundle": "Má soubory",
+  // "search.filters.applied.f.original_bundle_filenames": "File name",
+  // TODO New key - Add a translation
+  "search.filters.applied.f.original_bundle_filenames": "File name",
+  // "search.filters.applied.f.original_bundle_descriptions": "File description",
+  // TODO New key - Add a translation
+  "search.filters.applied.f.original_bundle_descriptions": "File description",
   // "search.filters.applied.f.itemtype": "Type",
   "search.filters.applied.f.itemtype": "Typ",
   // "search.filters.applied.f.namedresourcetype": "Status",
-  "search.filters.applied.f.namedresourcetype": "Stav",
+  "search.filters.applied.f.namedresourcetype": "Status",
   // "search.filters.applied.f.subject": "Subject",
   "search.filters.applied.f.subject": "Předmět",
   // "search.filters.applied.f.submitter": "Submitter",
   "search.filters.applied.f.submitter": "Předkladatel",
   // "search.filters.applied.f.jobTitle": "Job Title",
-  "search.filters.applied.f.jobTitle": "Pracovní pozice",
+  "search.filters.applied.f.jobTitle": "Název pracovní pozice",
   // "search.filters.applied.f.birthDate.max": "End birth date",
-  "search.filters.applied.f.birthDate.max": "",
+  "search.filters.applied.f.birthDate.max": "Datum narozdení do",
   // "search.filters.applied.f.birthDate.min": "Start birth date",
-  "search.filters.applied.f.birthDate.min": "",
+  "search.filters.applied.f.birthDate.min": "Datum narození od",
   // "search.filters.applied.f.supervisedBy": "Supervised by",
-  "search.filters.applied.f.supervisedBy": "Pod dohledem",
+  "search.filters.applied.f.supervisedBy": "Zkontrolováno",
```

Comment:
.

Reply:
@milanmajchrak opět bych to potřeboval vidět s většim kontextem, to je nějakej popis search filteru? To se týka "person" entity, závěrečné práce, nebo ještě něčeho jiného?